bigchaindb/core.py

```diff
@@ -136,11 +138,54 @@ def write_transaction(self, signed_transaction, durability='soft'):
         signed_transaction = deepcopy(signed_transaction)
         # update the transaction
         signed_transaction.update({'assignee': assignee})
+        signed_transaction.update({'assignment_timestamp': time()})
```

Comment:
Instead of using `time.time()` maybe we should use `bigchaindb.util.timestamp()`
It should not make any difference in this case but I feel it would be more consistent and this would not break if in the future we change how federation nodes get their time